MapboxAndroidDemo/src/main/AndroidManifest.xml

```diff
@@ -400,5 +394,23 @@
                 android:name="android.support.PARENT_ACTIVITY"
                 android:value="com.mapbox.mapboxandroiddemo.MainActivity"/>
         </activity>
+        <activity
+            android:name=".account.LandingActivity"
+            android:screenOrientation="portrait">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN"/>
+                <category android:name="android.intent.category.LAUNCHER"/>
+            </intent-filter>
+            <action android:name="android.intent.action.VIEW"/>
```

Comment:
yea, not really sure why it's being displayed that way on Github. On this branch, studio shows:

```java
  <activity
            android:name=".account.LandingActivity"
            android:screenOrientation="portrait">
            <intent-filter>
                <action android:name="android.intent.action.MAIN"/>
                <category android:name="android.intent.category.LAUNCHER"/>
            </intent-filter>
            <intent-filter>
                <action android:name="android.intent.action.VIEW"/>
                <category android:name="android.intent.category.DEFAULT"/>
                <category android:name="android.intent.category.BROWSABLE"/>
                <data
                    android:host="authorize"
                    android:scheme="mapbox-android-dev-preview"/>
            </intent-filter>
        </activity>
```
This is what we can use moving forward